app/controllers/campaigns_controller.rb

```diff
@@ -105,6 +129,24 @@ def associate_seller_distributor_params
     set_distributor
     @seller = Seller.find(params[:seller_id])
 
+    ret
+  end
+
+  def generate_campaign_gift_cards_params
+    params.require(:distributor_id)
+    params.require(:gift_card_amount_cents)
```

Comment:
I personally think the "cents" should be implied for anything on our API

Reply:
changed this to be "gift_card_amount"